layers/redis/Dockerfile

```
@@ -0,0 +1,11 @@
+ARG PHP_VERSION
+FROM bref/build-php-$PHP_VERSION AS ext
+
+RUN pecl install redis
```

That's it, so at the moment till redis ext gets removed from base image have to add force option:
```suggestion
RUN pecl install --force redis
```